Makefile

```diff
@@ -23,7 +23,11 @@ bazel-push-images:
 push: bazel-push-images
 
 bazel-test:
-	hack/dockerized "hack/bazel-fmt.sh && hack/bazel-test.sh"
+	if git diff --name-only | grep -q -v -e "md$$"; then \
```

Comment:
I think we should match ".md", including the dot. To make sure we are not skipping some non-doc files. run_cmd, bin_amd (made those up).

Reply:
Yes, I agree.
There is however a worse bug here.